Source/Core/CommandLineOptions.cs

```diff
@@ -2097,6 +2104,9 @@ be n (default 500)
                 only for monomorphic programs.
   /reflectAdd   In the VC, generate an auxiliary symbol, elsewhere defined
                 to be +, instead of +.
+  /laxFocus     Process foci in a bottom-up fashion. This way only generates
```

Comment:
/relaxFocus?

Reply:
Process foci in a bottom-up fashion, guaranteeing a linear number of splits are generated. 
The default (top-down) is more aggressive and may create an exponential number of splits.

Reply:
Please update the documentation also:

Process foci in a bottom-up fashion, guaranteeing a linear number of splits are generated.
The default (top-down) is more aggressive and may create an exponential number of splits.